src/components/capture/Create.tsx

```diff
@@ -77,8 +77,9 @@ function CaptureCreate() {
 
     // Form store
     const resetState = useEntityStore(fooSelectors.resetState);
-    const captureName = useEntityStore(fooSelectors.entityName);
```

Comment:
Out of curiosity, what was the motivation behind this renaming?

Reply:
Because eventually a lot of this needs to be shared between all forms and it does not matter if they are a capture or a materialization or something else. And want to make it easier to port over to something that just "creates an entity" in the future.